pants

```diff
@@ -2,6 +2,39 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+REPO_ROOT=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P)
+
+_interpreter_version_sentinel_file_path='.pants.d/.pants-python-version'
+
+function clean_if_interpreter_changed() {
+  if [[ -n "${_recursive_pants_shell_script_invocation:-}" ]]; then
+    return 0
+  fi
+
+  _python_version_file="${REPO_ROOT}/${_interpreter_version_sentinel_file_path}"
+
+  if [[ -f "$_python_version_file" ]]; then
+    _prev_run_python_version="$(cat "$_python_version_file")"
+    if [[ "$PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS" == "$_prev_run_python_version" ]]; then
```

Comment:
```suggestion
    if [[ "${PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS}" == "${previous_interpreter_constraints}" ]]; then
```

Reply:
I personally keep to the `"$VAR"` form for strings that are 100% the content of a variable dereference, and that's not what we do in https://github.com/pantsbuild/binaries, but that's not what we do in this repo so will make these changes -- thanks for pointing them out!